src/test/clojure/clara/test_java_facts.clj

```diff
@@ -0,0 +1,29 @@
+(ns clara.test-java-facts
+  (:require [clara.tools.testing-utils :as tu]
+            [clara.rules :as rules]
+            [clojure.test :refer [is deftest run-tests testing use-fixtures]])
+  (:import [clara.test.facts
+            BeanTestFact]))
+
+;; A test to demonstrate that a Pojo with indexed property accessors can be used as an alpha root in a session,
+;; see https://github.com/cerner/clara-rules/issues/446
+(tu/def-rules-test test-basic-rule
+  {:rules [kansas-rule [[[BeanTestFact
+                          (= ?locs locations)
```

Comment:
So would direct field access on `roadConditions` not work since it has no standard accessor?

Reply:
That's what I'd think yes.  A test that an actual invocation of the indexed accessor in a rule for the property without a standard accessor still works might be nice to have and wouldn't be hard to create but isn't crucial.  I realize it should still pass with this implementation, I just tend to prefer having tests be implementation agnostic where possible.

Reply:
added a test demonstrating that a compilation error is thrown when trying to use the standard accessor:
https://github.com/cerner/clara-rules/pull/449/commits/1a120651ad7f82c4ca32b6b46a0f60aaf84f94b0

Reply:
+1